### PR TITLE
support multi gpu per diloco worker in the run training script

### PR DESCRIPTION
**Pull request by samsja**

### CODE REVIEW

File: open_diloco/run_training.sh [hunk 1]
```diff
@@ -1,33 +1,47 @@
 #!/bin/bash
 
 ## example usage
-# ./run_training.sh 2 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
+# ./run_training.sh 2 1 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
```
Comment (Jackmin801) [inline open_diloco/run_training.sh:4]: Maybe have a 2 x 2 example and say that this uses 4 gpu total so its more clear that this is the intended arg usage.

**Review by Jackmin801 — APPROVED** [review]
nice! lgtm. minor comment on adding an extra example usage for clarity

File: open_diloco/run_training.sh [hunk 2]
```diff
@@ -1,33 +1,48 @@
 #!/bin/bash
 
 ## example usage
-# ./run_training.sh 2 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
+# ./run_training.sh 4 2 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
 # note that everything after the initial peer with will pass to all worker
```
Comment (samsja) [inline open_diloco/run_training.sh:5]: @Jackmin801 wdyt now

Comment (Jackmin801) [reply open_diloco/run_training.sh:5]: yea looks good

**Review by Jackmin801 — APPROVED** [review]

**Pull request merged**